### PR TITLE
release notes for May 20 release

### PR DESCRIPTION
**Pull request by tinaliang**

### CODE REVIEW

File: storage-resize-images/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+## Version 0.1.8
+
+fixed - fix bug where resized image did not keep orientation of original image (Issue #290
+fixed - release automation script for Github release (#302)
+fixed - clear cache so that image names can be reused (Issue #286) (#297)
+fixed - switched ImageMagick for Sharp library (#207)
```
Comment (rachelsaunders) [inline storage-resize-images/CHANGELOG.md:6]: These two items are also listed in the previous release.  I think that we need some clarity about what exactly was changed to have the same fixes mentioned again.

Comment (tinaliang) [reply storage-resize-images/CHANGELOG.md:6]: @laurenzlong can you provide more context here?

Comment (laurenzlong) [reply storage-resize-images/CHANGELOG.md:6]: only the first line should be in this release.

Comment (rachelsaunders) [reply storage-resize-images/CHANGELOG.md:6]: When you say "first line"... do you mean that only the first bullet (the one about orientation) should be in this release?

Comment (laurenzlong) [reply storage-resize-images/CHANGELOG.md:6]: yes.

File: storage-resize-images/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,10 @@
+## Version 0.1.8
+
+fixed - fix bug where resized image did not keep orientation of original image (Issue #290
```
Comment (rachelsaunders) [inline storage-resize-images/CHANGELOG.md:3]:
```suggestion
fixed - Resized images now maintain the same orientation as the original image. (Issue #290)
```

**Pull request merged**